### ISSUE TITLE
Expose class input to combo

### ISSUE THREAD

**Issue by Eralmidia** [IgniteUI/igniteui-angular#10187]
## Is your feature request related to a problem? Please describe.
While I understand you guys turning down #9832 in terms of supporting both single and an array of elements, I still feel the solution you guys have for single select is kinda half baked, in the sense that it displays checkboxes for single select (if anyything, this would be the job of a radio button, or better yet nothing at all, working like a select control). So I would like to propose someting which should be easer to implement, which will allow us to hide the checkboxes when using single select for the combo. The idea is simple, add a property/class to the combo for the combos which are single select, and then create some global CSS to hide the checkboxes. 

The issue with this is, of course, that the combo is "translated" into a dropdown which resides in the igx overlay. So I need a way to pass the class to this dropdown.

## Describe the solution you'd like
Basically exactly the panelClass functionality already present in the material select component: https://material.angular.io/components/select/examples#select-panel-class

**Comment by Lipata** [comment]
If you think that select is a solution for your case, we can try to create a demo with the select and templated first item, or tempted header that will contain input with pipe for filtering the list. There is one important difference between combo and select - the latter cannot be virtualized, because it is not integrated with igxFor. If you are binding it with lots of data, then this is not a proper solution.

**Comment by Eralmidia** [comment]
Yes, virtualization is a plus, so using the combo would be better. I guess this is not the end of the world, having checkboxes on single select, but your documentation do say: "Ignite UI for Angular bases its component designs on the Material Design Principles". And the material specifications clearly states:
![image](https://user-images.githubusercontent.com/12936286/136031782-2d977729-59fb-4005-8e42-c086a110e0ae.png)

**Comment by Lipata** [comment]
@Eralmidia we were following the [Material Selection](https://material.io/design/interaction/selection.html#properties) guide, where both touch and desktop were using checkboxes (for touch checkboxes they appear when the item is held). At that time we thought this was enough, but as we already agree about the single selection combo, following Material Guidelines is not a solution. I think that having radio buttons is also not a solution, because in the case of a single selection combo when the item is selected, then the drop-down is closed and the radio button will be unnecessary. Also visually it is not looking good.

While trying to implement the suggestion I gave you above (select with templated input), I found that you already tried to do that in the following [issue](https://github.com/IgniteUI/igniteui-angular/issues/8527). Knowing that the latter is not possible, for me what is left as a reasonable solution is #9832.
